working/1426-extension-types/feature-specification-implements-rep.md

```diff
@@ -0,0 +1,395 @@
+non-inline class `C` as a superinterface based on the class modifiers of `C`.
+
+*For instance, `inline class V implements B ...` does not give rise to a
+compile-time error because `B` is a class declared in a different library
+which is `sealed`, `final`, or `base`. The justification for this non-error
+is that the type `V` is a static device that allows us to work with an
+instance of `B` in a convenient way, it is not a mechanism that allows
+anything like "`V` is a subtype of `B`" to be a fact at run time (because
+`V` doesn't even exist at run time).*
+
+A compile-time error occurs if an inline class declaration _DV_ has two
+direct superinterfaces of the form `V1<T1 .. Tn>` and `V2<S1 .. Sm>` such
+that `V1` and `V2` resolve to the same declaration.
+
+*This rule is similar to a rule for non-inline classes that makes
+`class C implements B, B {}` an error.*
+
+Assume that an inline class declaration _DV_ has two superinterfaces,
+direct or indirect, of the form `V1<T1 .. Ts>` and `V2<S1 .. Ss>`. Assume
+that `V1` and `V2` both denote the same declaration. A compile-time error
+occurs if `Tj` and `Sj` are not the same type, for any `j` in `1 .. s`.
+This rule applies also in the case where one or both of the two types is a
+raw type, and the actual type arguments have been computed by instantiation
+to bounds, or by any other implicit mechanism.
+
+In this rule 'same type' is defined as in the section
+[Runtime type equality operator][] in the null safety specification.
+
+[Runtime type equality operator]: https://github.com/dart-lang/language/blob/main/accepted/2.12/nnbd/feature-specification.md#runtime-type-equality-operator
+
+*This means that `Tj` and `Sj` are the same type if the normalized form of
+both types are structurally identical up to names of type variables. For
+instance, `prefix.C<int>` and `C<int>` are the same type if `prefix.C` and
+`C` resolve to the same class declaration, and similarly for `int`;
+`FutureOr<Object>` is the same type as `Object` because of the
+normalization; and `void Function<X>(X)` and `void Function<Y>(Y)` are the
+same type based on alpha equivalence (renaming of type variables).*
+
+A similar rule applies for the case where _DV_ has two superinterfaces,
+direct or indirect, which are function types or record types *(in which
+hase the types _must_ be non-inline types)*: They must also be the same
+type.
+
+Let _DV_ be an inline class declaration named `V` with representation type
+`R` and assume that the `implements` clause of _DV_ includes the non-inline
+types `R1 .. Rk`. *We then have `R <: Rj` for each `j`, because anything
+else is an error.*
+
+Assume that `m` is a member which not declared by _DV_, and none of _DV_'s
+inline superinterfaces have a member named `m`, but one or more of the
+interfaces of `R1 .. Rk` has a member named `m`. A compile-time error
+occurs if there exist `j1` and `j2` in `1 .. k` and a member name `m` such
```

Comment:
We define `j1` and `j2`, and don't use them.
Do I miss something?
I understand what this paragraph saying, and it makes total sense.
Just nit picking.

Reply:
True, fixed, thanks again!